### PR TITLE
[MM-17924] Update message when an existing guest user is added to a single channel

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Updates the message when an existing guest user is added to a single channel to properly support plurals.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-17924

### CODE REVIEW

File: components/invitation_modal/invitation_modal_confirm_step_row.jsx [hunk 1]
```diff
@@ -55,7 +57,11 @@ export default class InvitationModalConfirmStepRow extends React.Component {
                     {guestBadge}
                 </div>
                 <div className='reason'>
-                    {invitation.reason}
+                    <FormattedMessage
```
Comment (jespino) [inline components/invitation_modal/invitation_modal_confirm_step_row.jsx]: This is not a valid approach, you are using the dynamically generated reason value to fill the `defaultMessage` but you are using the same key all the time, what means that is going to translate it wrongly. Or you pass all the data without translated and use here to build the FormattedMessage, or you pass the final translated version.

**Review by jespino — CHANGES_REQUESTED** [review]
You have to change the approach probably the simplest option is to generate 2 different reason messages based on the number of channels, and pass the generated reason message there.

**Comment by streamer45** [comment]
Looks like the proposed solution is not compatible with how translations are handled during build. mmjstool failing to extract the translation id.

**Comment by jespino** [comment]
@streamer45 Yes, you have to wrap the id inside the `t()` function to mark the text as something that you can translate.

**Review by jespino — APPROVED** [review]
LGTM

**Review by reflog — APPROVED** [review]
👍

**Review by srkgupta — APPROVED** [review]
Changes looks good and will be tested once the code is merged into master.

**Pull request merged**